### DISCUSSION TITLE
Private Repo and a Free Account

### DISCUSSION THREAD

**Discussion by mborl** [community/community#22941] [New to GitHub]
<p>Hello, I am new to GitHub and awaiting my student pack confirmation. Until then, one of my group members created a private repository that he has given me access to. If I create a branch or fork, will those become visible to the public since my account is still a regular, free account? I cannot let these files become public since they may only be worked on by my group for the class. Any help you can give is appreciated.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/mborl">@mborl</a>,</p>
<p>Welcome to GitHub! :slight_smile:</p>
<p>Yes, if you fork a private repository on an account that does not have a paid plan, the fork will remain private and will not count against your private repository quota. My understanding is that a branch of a private repo would also remain private, but I haven’t tested this out so can’t confirm this behaviour.</p>
<p>​If there’s anything else I can help you with, let me know; so happy to help further!</p>
<p>Kindest regards,<br>
Matthew</p>

**Comment by mborl** [comment]
<p>Thanks! How can I verify if the branch I create is private or not?</p>

**Comment by ghost** [comment]
<p>A public fork will look like this</p>
<p><img src="https://user-images.githubusercontent.com/10137/181073771-24822fab-e207-47f2-8739-b5926024dc41.png" alt="7957c7e5de71fac0545d015bd36a63ec" data-base62-sha1="6NXUbSWFDFoNQY9oqoLuxNNkTDW" width="482" height="98"></p>
<p>while a private fork will look similar to this:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fd313102fb15af1056dbb27ebeaae6a32476f0b9.png" data-download-href="/uploads/short-url/A7QbUU5WwtxYK1eHxJS4HnBuFMR.png?dl=1" title="c7218241ff2364296e6e4e9bbf3f6dfa" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181073772-8f1d0ee1-d010-4038-9fdf-c2145cc38f8f.png" alt="c7218241ff2364296e6e4e9bbf3f6dfa" data-base62-sha1="A7QbUU5WwtxYK1eHxJS4HnBuFMR" width="690" height="76" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fd313102fb15af1056dbb27ebeaae6a32476f0b9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">c7218241ff2364296e6e4e9bbf3f6dfa</span><span class="informations">736×82 7.38 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>where you can see it has a little locked icon and the word “Private” next to it.</p>
<p>Also, I checked your public GitHub profile and you don’t currently have any public repositories so if you <em>have</em> now created a branch of the existing private repository, it’s going to be private (otherwise it would show up as a public repository) :slight_smile:</p>
<p>​If there’s anything else I can help you with, let me know; so happy to help further!</p>
